assignment1/main.py

```diff
@@ -0,0 +1,56 @@
+print("")
+def first_prime_number(number):
+    def is_not_prime(number):
+        flag = False
+        for i in range(2, number):
+            if number % i == 0:
+                flag = True
+        return flag
+
+    while  is_not_prime(number):
+        number = number + 1
+    return number
+
+number = int(input("Enter a number: "))
+print("Problem 1: ")
+print(first_prime_number(number))
+print("")
+
+# Determine the twin prime numbers p1 and p2 immediately larger than the given non-null natural number n.
+# Two prime numbers p and q are called twin if q - p = 2.
+
+def twin_prime(number):
+    first = first_prime_number(number)
+    second = first_prime_number(first+1)
+    while second - first != 2:
+        first = second
+        second = first_prime_number(second+1)
+    if second - first == 2:
```

Comment:
This is straight out useless code. Your while stops when the difference is equal to 2.